.github/workflows/phpunit-oci.yml

```diff
@@ -0,0 +1,147 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: PHPUnit
```

Comment:
Since the app does not have migrations (does it run database queries), you could skip Oracle to save up to 10mins per CI run

Reply:
:+1: